details_energy.md

```diff
@@ -0,0 +1,177 @@
+
+#### Geothermal
+
+An additional XML file with Enhanced Geothermal Resources (EGS) is included with the standard GCAM input fileset, but is not included in the default configuration file. EGS expands the geothermal resource base significantly, albeit at higher costs, but is excluded from the default configuration due to the uncertainties about the future availability, effectiveness, and potential costs and risks of the technology.
+
+## Energy Transformation
+
+In energy transformation sectors, the output unit and input unit are EJ (per year), the price unit is 1975$ per GJ of output, and the subsector nest is used for competition between different fuels (or feedstocks). The competition between subsectors takes place according to a calibrated logit sharing function, detailed in [choice function](choice.html). Within the subsectors, there may be multiple competing technologies, where technologies typically represent either different efficiency levels, and/or the application of carbon dioxide capture and storage (CCS). The parameters relevant for technologies in GCAM are identified and explained in [energy technologies](en_technologies.html).
+
+In the schematic of the energy system depicted below, the energy transformation and distribution sectors include all sectors except for the resources (colored red) and the final demands (colored light blue).
+
+<img src="gcam-figs/energy_system_structure.jpg" width="900" height="650" /><br/>
+**Simplified schematic of the energy system in each region, showing the inter-sectoral flows of energy goods in GCAM.**
+{: .fig}
+
+### Electricity
+
+The nesting structure of the electric sector is shown in the figure below, with a focus on one repesentative technology.
+
+<img src="gcam-figs/elec_structure.png" width="900" height="200" /><br/>
+**Schematic showing the nesting structure of the electric sector, with levels for choices between fuels, technologies, and cooling systems. Note that this is a simplification of the actual structure used, which includes "pass-through" sectors, because GCAM sectors are only allowed two levels of nesting.**
+{: .fig}
+
+Details on the assumptions used in GCAM (e.g., cost, efficiency, capacity factors, etc.) is documented in [Muratori et al. 2017](energy.html#muratori2017). GCAM also includes the water inputs to electricity generation, in a third nest, as shown in the figure above. That is, within any thermo-electric generation technology, there is modeled competition between up to five different cooling system types; this is documented in the [water demand section](demand_water.html).
```

Comment:
With the ATB update we've moved past most of the Muratori assumptions.  So maybe the first sentence could be changed to "Details on the assumptions used in GCAM (e.g., cost, efficiency, capacity factors, etc.) is documented in [Inputs for Modeling Supply](inputs_supply.md#energy)."